flow/project.py

```diff
@@ -2114,11 +2116,13 @@ def _add_direct_cmd_arg_group(cls, parser):
             help="Manually specify all labels that are required for the direct command "
                  "to be considered eligible for execution.")
 
+    @deprecated(deprecated_in="0.6", removed_in="0.8")
```

Comment:
The versions specified in this line don't match the versions listed in the RuntimeError. It looks like this function has _already_ been removed. Am I missing something?

Reply:
@vyasr and I wanted to elevate the fact that this was already "removed."

I can go ahead and fully delete it in this PR?